index.html

```diff
@@ -1,156 +1,328 @@
-            <li><img src="Images_Background/JS-logo.svg" alt="JavaScript logo"><p class="language-name">JavaScript</p></li>
-            <li><img src="Images_Background/html-logo.svg" alt="HTML logo"><p class="language-name">HTML</p></li>
-            <li><img src="Images_Background/css-logo.svg" alt="CSS logo"><p class="language-name">CSS</p></li>
+            <li>
+              <img src="Images_Background/JS-logo.svg" alt="JavaScript logo" />
+              <p class="language-name">JavaScript</p>
+            </li>
+            <li>
+              <img src="Images_Background/html-logo.svg" alt="HTML logo" />
+              <p class="language-name">HTML</p>
+            </li>
+            <li>
+              <img src="Images_Background/css-logo.svg" alt="CSS logo" />
+              <p class="language-name">CSS</p>
+            </li>
           </ul>
-          <li class="bottom-line-language arrow-skills"><p>Frameworks</p><i class="fa fa-angle-down arrow-color"></i></li>
-          <li class="bottom-line-language arrow-skills"><p>Skills</p><i class="fa fa-angle-down arrow-color"></i></li>
-      </ul>
-    </section>
-  </main>  
-</body>
-</html>
\ No newline at end of file
+          <li class="bottom-line-language arrow-skills">
+            <p>Frameworks</p>
+            <i class="fa fa-angle-down arrow-color"></i>
+          </li>
+          <li class="bottom-line-language arrow-skills">
+            <p>Skills</p>
+            <i class="fa fa-angle-down arrow-color"></i>
+          </li>
+        </ul>
+      </section>
+
+      <!------------------------>
+      <!-- CONTACT ME SECTION -->
+      <!------------------------>
+
+      <section id="contact-me">
+        <h2 class="contact-me-title">Contact me</h2>
+        <p class="contact-me-paragraph principal-margin">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it!
+        </p>
+        <form action="https://formspree.io/f/mknayrpk" method="post" class="principal-margin">
+          <input
+            type="text"
+            id="name"
+            class="form-input form-input-width"
+            name="user-name"
+            maxlength="50"
+            placeholder="Hernán Güemes"
+            required
+          />
+          <input
+            type="email"
+            class="form-input form-input-width"
+            id="mail"
+            name="user-email"
+            placeholder="hernanguemes@hotmail.com"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
+            class="form-input-width textarea-size"
+            maxlength="500"
+            placeholder="Write your message here."
+            required
+          ></textarea>
+          <button type="submit" class="see-project-button contact-form-button">
+            Get in touch
+          </button>
```

Comment:
- [ ] I noticed that you are using `border-radius: 5px` . Please change it to `8px` as given in the Figma design guidelines 🤓⛏
- [ ] Similarly, the color given in the design guidelines is `#6070FF` not `#396df2`. Please fix this as well 💪
- [ ] For `hover` effect, please consider adding `box-shadow` also as given.
- [OPTIONAL] I think it would be better to use `cursor: pointer` when  a user interacts with the button. What do you think? 💭😅